src/parser.nim

```diff
@@ -189,6 +190,7 @@ proc parseTweet(js: JsonNode): Tweet =
     replyId: js{"in_reply_to_status_id_str"}.getId,
     text: js{"full_text"}.getStr,
     time: js{"created_at"}.getTime,
+    source: js{"source"}.getStr.split(">")[1].split("<")[0],
```

Comment:
Just in case source ever changes format or is empty we need to extract it in a way that won't crash on errors:
```nim
str.substr(str.find('>') + 1, str.rfind('<') - 1)
```

You can put this in a `getSource` proc in parserutils below `getTombstone`